src/components/DistanceRequest.js

```diff
@@ -80,6 +80,36 @@ function DistanceRequest({transactionID, transaction, mapboxAccessToken}) {
     const numberOfWaypoints = _.size(waypoints);
     const lastWaypointIndex = numberOfWaypoints - 1;
 
+    const waypointsData = _.filter(
+        _.map(waypoints, (waypoint, key) => {
+            if (waypoint.lng === undefined || waypoint.lat === undefined) {
```

Comment:
NAB: We could simplify to `if (!waypoint.lng || !waypoint.lat)`

Reply:
it's possible that either longitude and latitude is `0` 🤔

Reply:
Oh true good point